.github/workflows/package.yml

```diff
@@ -403,3 +403,54 @@ jobs:
           allowUpdates: true
           updateOnlyUnreleased: true
           artifacts: "${{ env.PACKAGE_DIR }}.tar.gz"
+
+  build-deb:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      attestations: write
+      id-token: write
+    env:
+      BUILD_DIR: Dist
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get install -y --no-install-recommends cmake make g++ libpcap-dev
+
+      - name: Build project
+        run: |
+          mkdir -p "${{ env.BUILD_DIR }}"
+          cd "${{ env.BUILD_DIR }}"
+          cmake ..
+          make -j$(nproc)
+          make install DESTDIR=$PWD/install-root
```

Comment:
Instead of installing PcapPlusPlus here, maybe we can build the `deb` file, install it, and add another step to test that it works?